ocs_ci/utility/utils.py

```diff
@@ -4427,3 +4429,71 @@ def archive_ceph_crashes(toolbox_pod):
     """
     log.info("Archiving all ceph crashes")
     toolbox_pod.exec_ceph_cmd("ceph crash archive-all")
+
+
+def add_time_report_to_email(session, soup):
+    """
+    Takes the time report dictionary and converts it into HTML table
+    """
+    table_body_html = ""
+
+    data = GV.TIMEREPORT_DICT
+    sorted_data = dict(
+        sorted(data.items(), key=lambda item: item[1]["total"], reverse=True)
+    )
+    count = 1
+    for test, values in sorted_data.items():
+        if count > 5:
+            break
+        table_body_html += f"""
+        <tr>
```

Comment:
Can we re-write it via using jinja template:
https://url.corp.redhat.com/629bd6b